Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# 3rd Small Improvements

## Changelog:
- Troll emperors are called God Kings/Queens.
- The Horde Invasion ends in the year 603 now.
- Mentioned the Uninvited Guest as a goblin's evil god and the Everlasting Party as their afterlife (localization sometimes seen in events).
- Added new religious symbol names (localization sometimes seen in events).
- Troll Loa pantheons have the Adorcism tenet.
- Troll Loa pantheons, Agamagganic and Goldrinnism got Blood Feast tenet that combines Human Sacrifice and Cannibalism.
- Goldrinnism has the Natural Primitivism.
- Possessed trait is a Shath'gral and Doombringer virtue.
- The Lunatic trait is a Shath'gral virtue.


## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log`


# How to test:
Play the game, test each feature. Don't forget to use console commands.

- `set_date 602` to set the year to 602.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/648/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here are the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).
